tests/test_pipelines_ner.py

```diff
@@ -190,6 +189,329 @@ def _test_pipeline(self, nlp: Pipeline):
             ],
         ]
 
+        ungrouped_inputs_all_scores = [
```

Comment:
This is a lot of waste of space for something we don't really want to read so I'd group all the arrays back on one line and disable the styling for this list (place a line with `# fmt off` before and a line with `# fmt off` after).

Or this could be stored as a fixture (in the fixtures folder) after saving it to json?